bin/oref0-pump-loop.sh

```diff
@@ -212,6 +212,10 @@ function smb_reservoir_before {
     echo -n " is within 90s of current time: " && date +'%Y-%m-%dT%H:%M:%S%z'
     if (( $(bc <<< "$(to_epochtime $(cat monitor/clock-zoned.json)) - $(epochtime_now)") < -55 )) || (( $(bc <<< "$(to_epochtime $(cat monitor/clock-zoned.json)) - $(epochtime_now)") > 55 )); then
         echo Pump clock is more than 55s off: attempting to reset it and reload pumphistory
+	        # Check for bolus in progress
+                smb_verify_status && \
```

Comment:
Probably should use a couple of `if` clauses here: I don't know what the `command1 && # comment \n command2` behavior is.  Also, looks like your indentation doesn't match the surrounding code.

Reply:
Consolidated comments, and fixed indentation...

https://xkcd.com/1513/